src/data/move.ts

```diff
@@ -10280,6 +10295,8 @@ export function initMoves() {
       .ignoresSubstitute()
       .attr(RepeatMoveAttr)
       .edgeCase(), // incorrect interactions with Gigaton Hammer, Blood Moon & Torment
+    // Also has incorrect interactions with move-calling moves (Mirror Move, Copycat/Mimic, Metronome)
+    // and Dancer that will need to be cleaned up separately
```

Comment:
```suggestion
      // incorrect interactions with Gigaton Hammer, Blood Moon & Torment
      // Also has incorrect interactions with move-calling moves (Mirror Move, Copycat/Mimic, Metronome)
      // and Dancer that will need to be cleaned up separately
      .edgeCase(),
```
Since #3380 was merged you should probably be able to fix that interaction now.